<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageModifierAppClass</class>
 <widget class="QMainWindow" name="ImageModifierAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageModifierApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="imgHLayout" stretch="1,1">
      <item>
       <widget class="QLabel" name="imgOriginal">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>NO IMAGE FOUND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imgTransformed">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>NO IMAGE FOUND</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuTransform">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Transform</string>
     </property>
     <addaction name="actionReset"/>
     <addaction name="separator"/>
     <addaction name="actionBlur"/>
     <addaction name="actionErode"/>
     <addaction name="actionDilate"/>
     <addaction name="actionOpening"/>
     <addaction name="actionClosing"/>
     <addaction name="actionCannyFilter"/>
     <addaction name="actionDetectEdges"/>
     <addaction name="actionDetectVertices"/>
     <addaction name="actionDetectObjects"/>
     <addaction name="separator"/>
     <addaction name="actionOpenToolbox"/>
    </widget>
    <addaction name="menuTransform"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionErode">
   <property name="text">
    <string>Erode</string>
   </property>
  </action>
  <action name="actionDilate">
   <property name="text">
    <string>Dilate</string>
   </property>
  </action>
  <action name="actionOpening">
   <property name="text">
    <string>Opening</string>
   </property>
  </action>
  <action name="actionClosing">
   <property name="text">
    <string>Closing</string>
   </property>
  </action>
  <action name="actionCannyFilter">
   <property name="text">
    <string>Canny Filter</string>
   </property>
  </action>
  <action name="actionDetectEdges">
   <property name="text">
    <string>Detect Edges</string>
   </property>
  </action>
  <action name="actionDetectVertices">
   <property name="text">
    <string>Detect Vertices</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionOpenToolbox">
   <property name="text">
    <string>Open Toolbox</string>
   </property>
  </action>
  <action name="actionDetectObjects">
   <property name="text">
    <string>Detect Objects</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ImageModifierApp.qrc"/>
 </resources>
 <connections/>
</ui>
